codec-dns/src/test/java/io/netty/handler/codec/dns/package-info.java

```diff
@@ -1,21 +0,0 @@
-/*
- * Copyright 2012 The Netty Project
- *
- * The Netty Project licenses this file to you under the Apache License,
- * version 2.0 (the "License"); you may not use this file except in compliance
- * with the License. You may obtain a copy of the License at:
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations
- * under the License.
- */
-
-/**
- * A DNS client that queries a server and checks if query information and
- * responses are valid to ensure codec is correct.
- */
-package io.netty.handler.codec.dns;
```

Comment:
Maybe doesn't belong in this PR, but this is a duplicate of the `package-info.java` in the `main` directory tree.